mixes/containers/mix.js

```diff
@@ -7,25 +7,32 @@ const { loadMix, saveMix } = require('../actions')
 class MixContainer extends React.Component {
   render () {
     const { mix, isLoading, isSaving, error, saveMix } = this.props
+    if (!mix) { return null }
```

Comment:
yes views should be defensive. assume they might be called with any amount of incomplete info and still do something sensible, so you don't need to worry about how the actions progress.